x-pack/plugins/maps/public/layers/tooltips/es_tooltip_property.ts

```diff
@@ -64,12 +68,19 @@ export class ESTooltipProperty implements ITooltipProperty {
     );
   }
 
-  async getESFilters(): Promise<PhraseFilter[]> {
+  async getESFilters(): Promise<Filter[]> {
     const indexPatternField = this._getIndexPatternField();
     if (!indexPatternField) {
       return [];
     }
 
-    return [esFilters.buildPhraseFilter(indexPatternField, this.getRawValue(), this._indexPattern)];
+    const value = this.getRawValue();
+    if (value == null) {
```

Comment:
`==` relies on type coercion. maybe `value === null || typeof value == 'undefined'` instead.

Reply:
I would like to leave the check as `== null`. This is a pretty standard way to check for null, for example, from https://medium.com/javascript-in-plain-english/how-to-check-for-null-in-javascript-dffab64d8ed5 "That means checking for null is one of the few times in JavaScript that using == is recommended, while otherwise === is generally recommended." Additionally, this null check is used all over the Kibana code base. Just search for `== null` to see its usage.